### PR TITLE
Improve the clarity of the readme markdown file.

### PR DESCRIPTION
**Pull request by georgeliao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ you would need to override the systemd service with the following setting:
 
 ```conf
 [Service]
-Environment="MULTIPASS_BLUEPRINTS_URL=https://github.com/USERNAME/multipass-blueprints/archive/refs/heads/BRANCH_NAME.zip"
+Environment="MULTIPASS_BLUEPRINTS_URL=https://github.com/canonical/multipass-blueprints/archive/refs/heads/<BRANCH_NAME>.zip"
```
Comment (ricab) [inline README.md:59]: This wouldn't be valid for a fork (which a contributor without write access to the repository needs to use).

**Review by ricab — COMMENTED** [review]
Hey @georgeliao, thanks for improvement. I am fine with the clarificatoin you added in the text, but I am not convinced by the update to the link.

File: README.md (hunk 1 above)
Comment (georgeliao) [reply README.md:59]: Indeed, I missed the context a bit. 
USERNAME can mislead to the actual github user name of ours for the folk in Canonical. Meanwhile, USERNAME/organization_name seems to be the right name for that part of the URL. So I also do not have a good way to disambiguate this. Maybe I can change it back and just add<> to emphasize it is a variable?

Comment (ricab) [reply README.md:59]: Right. How about `<REPO_OWNER>`?

Comment (georgeliao) [reply README.md:59]: Sounds better.

**Review by ricab — APPROVED** [review]
Looks good, thanks!

**Pull request merged**